meeting-minutes/team-minutes/oct26-team-minutes.md

```diff
@@ -8,6 +8,22 @@
 - [Katrina] Talk about last week's client meeting to make sure everyone's caught up on the notes (i.e the meeting will be on Google Meets instead of Zoom, possible ways to deploy the website, what we'd like to show for Oct. 27th's client meeting) 
 
 ## Detailed Notes:
+- All members got the front end setup completed
+- All members got the back end setup completed
+- All members installed nodejs
+- App is viewable for everyone, both front end and back end
+- New setup for how branches work
+- New setup for how issues work
```

Comment:
It might be useful in the future to put a little bit more detail of what the 'new setup' looks like so that if we ever need to refer back to the minutes from today's meeting, we can pinpoint exactly what we talked about (i.e the naming convention for branches now includes the issue number the task has been associated with. In the future we might change the convention though, so it'd be nice to have it in writing to keep track of what's changed, and the latest standards).

Reply:
Agreed!